socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -47,6 +47,21 @@ dalvik-jit-code-cache
 dalvik-LinearAlloc
 data@app@org\.mozilla\.f.*\.apk@classes\.dex
 DebugBreak
+_dispatch_client_callout
+_dispatch_continuation_pop
+_dispatch_lane_barrier_sync_invoke_and_complete
+_dispatch_main_queue
+_dispatch_once_callout
+_dispatch_queue_invoke
+_dispatch_queue_override_invoke
+_dispatch_queue_serial_drain
+_dispatch_root_queue
+_dispatch_source_invoke
+_dispatch_source_latch_and_call
+_dispatch_sync_invoke_and_complete_recurse
+_dispatch_sync_wait
+_dispatch_worker_thread
+_dispatch_workloop_worker_thread
```

Comment:
Are there `_dispatch_*` functions we want to keep? If not, we can just use `_dispatch` here because the lines in this file are all prefix matches.

Reply:
I looked around and there are at least some functions which we don't want to ignore, so for now I'd keep them separate.